tests/visual/percy-storybook/DashTable.percy.tsx

```diff
@@ -64,7 +96,7 @@ storiesOf('DashTable/With Data', module)
         ]}
     />));
 
-const columns = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j']
+const columnsA2J = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j']
```

Comment:
Does `A2J` stand for `Alex Johnson was here`? 🤔

Reply:
haha I almost called it `columnsAJ` but thought that might be too selfish. It's supposed to be just `'a' -> 'j'`